code/modules/mob/living/silicon/robot/modules/module_clerical.dm

```diff
@@ -37,7 +37,7 @@
 		/obj/item/tray/robotray,
 		/obj/item/reagent_containers/borghypo/service
 	)
-	emag = /obj/item/reagent_containers/food/drinks/bottle/small/beer
+	emag = /obj/item/melee/energy/sword ///obj/item/reagent_containers/food/drinks/bottle/small/beer , this is literally a beer bottle, plus so we dont make the standard module stans angry, we are putting the esword to this guy
```

Comment:
No. The service borg isn't meant to be a direct fighter when emagged. Their whole shtick is poisoning drinks or other types of subterfuge. They have a shit direct weapon to discourage trying to get into fights directly. Even if you wanted to upgrade it why an e-sword that's just op.

Reply:
even though its weird just getting a beer bottle, i added it back

Reply:
I think its cause you can break it an use it as a melee weapon, but again not supposed to be actually effective as a direct melee fighter so it makes sense. Plus beer for a service bot.